src/debug/debugSession.ts

```diff
@@ -174,7 +184,10 @@ export class DebugSession implements IDebugSession {
         // Find the debug port to attach.
         const portInfo = await this.debugProvider.resolvePortsFromContainer(this.kubectl, targetPod, targetContainer);
         if (!portInfo || !portInfo.debugPort) {
-            vscode.window.showErrorMessage("Cannot resolve the debug port to attach.");
+            const answer = await vscode.window.showErrorMessage("Cannot resolve the debug port to attach, please refer to the wiki for the usage.", "Open in Browser");

```

Comment:
It seems like the `const answer = …; if (open in browser) { … }` is a repeated pattern - consider extracting a helper method.